packages/webapp/src/pages/Logs/Search.tsx

```diff
@@ -0,0 +1,114 @@
+import { LeftNavBarItems } from '../../components/LeftNavBar';
+import DashboardLayout from '../../layout/DashboardLayout';
+import { useStore } from '../../store';
+import Info from '../../components/ui/Info';
+import { Loading } from '@geist-ui/core';
+import { useSearchLogs } from '../../hooks/useLogs';
+import * as Table from '../../components/ui/Table';
+import { getCoreRowModel, useReactTable, flexRender } from '@tanstack/react-table';
+
+import { MultiSelect } from './components/MultiSelect';
+import { columns, statusDefaultOptions, statusOptions } from './constants';
+import { useEffect, useState } from 'react';
+import type { SearchLogsState } from '@nangohq/types';
+import Spinner from '../../components/ui/Spinner';
+import { OperationRow } from './components/OperationRow';
+
+export const LogsSearch: React.FC = () => {
+    const env = useStore((state) => state.env);
+
+    // Data fetch
+    const [states, setStates] = useState<SearchLogsState[]>(statusDefaultOptions);
+    const [hasLogs, setHasLogs] = useState<boolean>(false);
+    const { data, error, loading } = useSearchLogs(env, { limit: 20, states });
+
+    const table = useReactTable({
+        data: data ? data.data : [],
+        columns,
+        getCoreRowModel: getCoreRowModel()
+    });
+
+    useEffect(() => {
+        if (!loading) {
+            // We set this so it does not flicker when we go from a state of "filtered no records" to "default with records"...
+            // ...to not redisplay the empty state
+            setHasLogs(true);
+        }
+    }, [loading]);
+
+    if (error) {
+        return (
+            <DashboardLayout selectedItem={LeftNavBarItems.Logs} marginBottom={60}>
+                <Info color={error.error.code === 'feature_disabled' ? 'orange' : 'red'} classNames="text-xs" size={20}>
```

Comment:
This text is super small -- haven't checked the design but based on the app relative to everything else it seems too small IMO.

Reply:
yes agreed, I have already mentioned that to Gabrielle